src/gui/qgstemporalcontrollerwidget.cpp

```diff
@@ -281,8 +281,13 @@ void QgsTemporalControllerWidget::togglePause()
 
 void QgsTemporalControllerWidget::updateTemporalExtent()
```

Comment:
Sorry, but I always get lost about what 'updateTemporalExtent' of a temporal widget is doing:

- A is it recalculating the min/max TimeExtent (including the duration) of all the layers?
- B is it (needed for) updating the slider limits/steps/ticks?
- C is it setting current temporal filter?

Header comment says: "Updates the navigation temporal extent." which is rather confusing to me.
(I think it is B and C while intuitive I think A)

Reply:
I agree, I think this API is confusing too